docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -37,35 +60,81 @@ gcloud compute scp <master-node-instance>:/var/run/kubernetes/client-ca.crt /tmp
 
```

Comment:
suggest update this section to current official scp command, such as:
```
gcloud beta compute ssh --zone "us-west2-a" "doctests-worknode"  --project "workload-controller-manager"
```

Reply:
Updated. Thanks!

Reply:
you may need replace ""doctests-worknode"  and "workload-controller-manager" with "worker node machine name" and "gce project name"